java/src/net/razorvine/pickle/PickleUtils.java

```diff
@@ -367,6 +376,15 @@ public static String decode_unicode_escaped(String str) {
 						c2=(char)Integer.parseInt(""+h1+h2+h3+h4, 16);
 						sb.append(c2);
 						break;
+					case 'n':
+						sb.append('\n');
+						break;
+					case 'r':
+						sb.append('\r');
+						break;
+					case 't':
+						sb.append('\t');
+						break;
```

Comment:
This duplication sucks but I figured it was simpler than adding some other function and contorting the case statements around to be ok with it, but YMMV